core/DataAccess/LogAggregator.php

```diff
@@ -169,6 +169,11 @@ class LogAggregator
      */
     private $allowUsageSegmentCache = false;
 
+    /**
+     * @var \Piwik\Tracker\Db|\Piwik\Db\AdapterInterface|\Piwik\Db
+     */
+    private $readerDb;
```

Comment:
btw @diosmosis I suppose this one isn't needed?

Reply:
removed it